docs/member/fixed-income.md

```diff
@@ -0,0 +1,128 @@
+
+**iii. Investment Calculator:**  Check the market price, add your quantity and calculate the total investment using the Investment Calculator. You can check the total investment and payout along with the maturity date from here. Click on Buy Now to buy the bond.
+
+[![Fixed Income](imgs/fixed_income_investment_calculator.png "Click to Enlarge or Ctrl+Click to open in a new Tab")](imgs/fixed_income_investment_calculator.png)
+
+**iv. Payment Cycle:**  This section will display the payment cycle of the security, this includes the price considered, interest rate and principal + interest.
+
+[![Fixed Income](imgs/fixed_income_payment_cycle.png "Click to Enlarge or Ctrl+Click to open in a new Tab")](imgs/fixed_income_payment_cycle.png)
+
+**v. Orders:** In the orders section you can sell any bond, search for your orders or get the complete order details.
+
+
+* **Sell Any Security:** To sell a security you will need details such as the ISIN, Settlement Date, Quantity, Settlement Amount or Principal Amount & Accrued Interest. Before you sell a security, ensure your KYC is completed.
+
+* **Search Bar:** Search any order using the search bar.
+
+* **Filters & Sort:** Filter your orders by Issuer, IP Frequency, Taxation Criteria, Type & Status. You can Sort your orders by applying the Yield or Safety options.
+
+* **All Orders:** Check all your orders in this section. This includes the Issuer & security name, Settlement amount, Maturity Date, Status, Deal Sheet and more.
+
+[![Fixed Income](imgs/fixed_income_orders.png "Click to Enlarge or Ctrl+Click to open in a new Tab")](imgs/fixed_income_orders.png)
+
+**vi. Portfolio:** This is your Fixed Income portfolio you can sell any security, search or filter the securities and view the complete portfolio details.
+
+
+* **Sell Any Security:** To sell a security you will need details such as the ISIN, Settlement Date, Quantity, Settlement Amount or Principal Amount & Accrued Interest. Before you sell a security, ensure your KYC is completed.
+
+* **Search Bar:** Search any order using the search bar.
+
+* **Filters & Sort:** Filter your orders by Issuer, IP Frequency, Taxation Criteria, Type & Status. You can sort your orders by applying the Yield or Safety options.
+
+* **Portfolio:** This is your fixed income portfolio section where the security details, purchase date, Interest Payment Date, Total Cost Price, Total Current Value and Action are displayed.
+
+[![Fixed Income](imgs/fixed_income_portfolio.png "Click to Enlarge or Ctrl+Click to open in a new Tab")](imgs/fixed_income_portfolio.png)
+
+**vii. Manage Accounts:** Manage your Fixed Income details such as add your Bank Account, Demat Details & KYC Information.
+
+## 4. How To Buy a security?
+
+It's crucial to update your KYC, bank information, and demat account information in the Manage Account section in the sidebar navigation before you Buy a security.
```

Comment:
`It's crucial to update your KYC`

change to

`It is crucial...`

Reply:
`Manage Account section in the sidebar navigation before you Buy a security.`

change to

`Manage Account section from the sidebar navigation before you buy a security.`

Reply:
Done